projects/plugins/backup/src/js/components/Admin.js

```diff
@@ -1,17 +1,51 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
Since we're using WordPress elements for other parts of the interface, what would you say about relying on the abstractions of `@wordpress/element` for all those things, so we can rely on the WordPress bundle and not have to worry about React at all?

Reply:
Following up on this in #20343.